R/ard_continuous.R

```diff
@@ -178,7 +180,7 @@ ard_continuous <- function(data,
 #' data <- data.frame("ard_x" = 1)
 #'
 #' cards:::.check_no_ard_columns(data)
-.check_no_ard_columns <- function(x, call = parent.frame(), exceptions = "...ard_dummy_for_counting...") {
+.check_no_ard_columns <- function(x, call = get_cli_abort_call(), exceptions = "...ard_dummy_for_counting...") {
```

Comment:
Can we remove the call argument? (gah, sorry for repeating this, but want to flag them in case we do remove)